core_lib/src/managers/selectionmanager.cpp

```diff
@@ -150,12 +150,12 @@ void SelectionManager::adjustSelection(const QPointF& currentPoint, const QPoint
     switch (mMoveMode)
     {
     case MoveMode::MIDDLE: {
-        QPointF newOffset = currentPoint - offset;
+        QPointF newOffset = currentPoint - mAlignToAxisStartPosition;
```

Comment:
Although the change itself is fine, I find the variable misleading now. Maybe we should rename to something more fitting?

How about mDragOrigin?

and in header...

/// The point from where the dragging will be based of inside the selection area.
/// Not to be confused with the selection origin
void setDragOrigin(const QPointF point) { mDragOrigin = point; }